cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/CaReconciler.java

```diff
@@ -594,5 +594,5 @@ Future<Void> maybeRemoveOldClusterCaCertificates() {
     /**
      * Helper class to pass both Cluster and Clients CA as a result of the reconciliation
      */
-    protected record CaReconciliationResult(ClusterCa clusterCa, ClientsCa clientsCa) { }
+    public record CaReconciliationResult(ClusterCa clusterCa, ClientsCa clientsCa) { }
```

Comment:
Why does this need to be public?

Reply:
We return it via a public method. So yes, it should be public.